frontend/components/utils/FormatCurrency.vue

```diff
@@ -1,10 +1,12 @@
 <template>
-    <span>
-        {{ sign }}<animated-number v-if="isValidNumber" :value="value" :decimal-places="decimals" /><span
-            v-else-if="value"
-            >NaN</span
-        ><span class="uppercase"> {{ currency }}</span>
-    </span>
+    <span
+        >{{ sign
+        }}<span v-if="isValidNumber"
+            ><animated-number v-if="value > 0.01" :value="value" :decimal-places="decimals" /><span v-else>{{
+                formattedNumber
+            }}</span></span
```

Comment:
Why would you want to disable animation and reimplement formatting if you can just properly determine number of decimals. Not even here, but inside `AnimatedNumber` itself?

Reply:
Because I do not just change the decimal point. When the number becomes too small it gets formatted like this: 1e-13 DAI. 

This kind of formatting sadly does not work with the Animated Number library. 

However I can do it for numbers that do not get formatted in this special way.

Reply:
> When the number becomes too small it gets formatted like this: 1e-13 DAI

We should not display those kind of numbers, as we can not expect from users to understand mathematical notation. Instead, we should just change the number of decimal points or even better: show something like `less than 0.01 DAI`

Reply:
Do you think 0.01 is a good cut of point for this? I would maybe display the decimal places until `0.00001` or maybe even later.